jwst/datamodels/schemas/keyword_channel.schema.yaml

```diff
@@ -9,5 +9,5 @@ properties:
           channel:
             title: NIRCAM or MIRI channel name.
             type: string
-            enum: [LONG, SHORT, '12', '34', '1', '2', '3', '4', ANY, N/A]
+            enum: [LONG, SHORT, '1234', '123', '234', '12', '34', '1', '2', '3', '4', ANY, N/A]
```

Comment:
If the enum list is being updated in this sub-schema, then it should be updated in the core schema too.

Reply:
Or were those additional values already in the core schema?

Reply:
Yup, these were already in the core schema, now all three match.

Reply:
They appear to be in the core schema already.

Reply:
However, the same change needs to be made to `keyword_pchannel`.

Reply:
I am probably missing something - but for the NIRCAM or MIRI channel name you also need MEDIUM (for MIRI).

Reply:
@jemorrison Isn't `MEDIUM` a `band` value? `channel` for MIRI is a number, right?

Reply:
My bad we use them interchangeable on the team. You are right  for the pipeline it is a number.

Reply:
(These are added to `keyword_pchannel` in this PR)